docs/001_develop/03_client-capabilities/011_criteria/02_criteria-builder.mdx

```diff
@@ -0,0 +1,103 @@
+---
+title: 'Criteria builder'
+sidebar_label: 'Criteria builder'
+id: criteria-builder
+keywords: [foundation, ui, comms, criteria, filtering]
+tags:
+  - connected
+  - ui
+  - comms
+  - criteria
+  - filtering
+---
+
+The `CriteriaBuilder` is a utility class designed to build objects used in creating query criteria in a Groovy expression format. These criteria can be used in requests to backend services.
+
+## Methods
+
+:::info
+
+The examples for these methods use a simple expression which checks `TRADE_ID` equals a certain value.
+
+```ts
+const tradeIdEqualsExpression = new ExpressionBuilder()
+  .withField('TRADE_ID')
+  .withValue('TR1234')
+  .withSerialiser(Serialisers.EQ)
+  .build()
+```
+:::
+
+
+<table>

```

Comment:
![Screenshot 2024-11-29 at 11 19 22](https://github.com/user-attachments/assets/a9772d9b-e625-4a93-a7d3-1b1597e9506d)

Nitpicky but there is something wrong with the formatting of the table header, it isn't coloured like everything else.

ALso can you format the code example lines like normal (left indented with space above and below the example) as this allows it to render in other markdown viewers like github too

Reply:
Fixed